### ISSUE TITLE
feature request: multiple saves

### ISSUE THREAD

**Issue by hollo6** [Jacudibu/PRUNner#9]
It would be very useful to have multiple saved states for a planet (or even the empire if it's easier to implement), so we could make and save/load incremental expansion plans.

**Comment by Jacudibu** [comment]
Yeah, a little dropdown for different (named) plans & stages would really be neat.
For now you can add the same planet multiple times and it will work just fine. :)

**Comment by hollo6** [comment]
wow, I didn't even think about trying that. it's a great idea, will do that for now. :)
